repaygithub/cactus
# CACTUS-722 :: Icon Export Updates

Ticket: https://repayonline.atlassian.net/browse/CACTUS-722

Credit to @wilysword for doing the majority of the actual work here. I just cherry-picked his commit and cleaned things up a bit, then updated cactus-icons usage elsewhere.

I think I caught all of the usages & updated them accordingly, but let me know if you notice anything I missed.

### Testing
1. Check out this branch and run `yarn icons cleanup && yarn icons build`.
2. Run `yarn icons start` and play around with the stories. Make sure everything looks good.
4. Run ` yarn web start` and check out the icon-related features in the `ConfirmModal`, `List` and `SplitButton` stories. Make sure they all work as expected.
5. Run `cd examples/theme-components && yarn start`. Bring the app up in your browser and check out the `IconButton` and `Icon` example pages to make sure they're working.
